s3/replication/common/tests/system/config/config.yaml

```diff
@@ -4,6 +4,8 @@ s3_region: us-west2
 source_bucket_name: sourcebucket
 target_bucket_name: targetbucket
 object_name_prefix: foo_
+object_tag_name: user-tag
+object_tag_value: get-obj-tagging
```

Comment:
may be use `user-tag-value` as value part.

Reply:
done